src/index.spec.ts

```diff
@@ -1,12 +1,180 @@
+import { valid } from 'joi';
 import {
   isInteger,
   toLowerCase,
   removeDuplicatesFromArray,
   createRandomProduct,
   getStarWarsPlanets,
-  createProduct 
+  createProduct,
 } from './index';
+import fetch from 'node-fetch';
+import { mocked } from 'ts-jest/utils';
 
-test('Testing time!', () => {
-  console.log('welcome');
+ describe('isInteger', () =>{
+  test('Return true if we pass a number', () =>{
+    expect(isInteger(8)).toBe(true);
+  });
```

Comment:
Inside a describe statement, it's more readable to use it() instead of test(), and starting with the string `should` in lowercase. For example:

```
it('should return true if receives an integer number')
it('should return false if receives a string')
it('should return false if receives a null value')
```

Reply:
You could have added a test for a decimal number.